src/components/notion-blocks/video.tsx

```diff
@@ -0,0 +1,32 @@
+import styles from '../../styles/notion-block.module.css'
+import React from 'react';
+import YouTube, {YouTubeProps } from 'react-youtube';
+
+
+const Video=({block})=> {
```

Comment:
```suggestion
const Video=({ block })=> {
```
`{}` の内側にはスペースを1個お願いします。